perl/lib/Gherkin/TokenMatcher.pm

```diff
@@ -238,6 +238,9 @@ sub match_StepLine {
 
 sub match_DocStringSeparator {
     my ( $self, $token ) = @_;
+    if ($token->is_eof) {
+        return;
```

Comment:
Other implementations return `true`/`false` in `_match_DocStringSeparator`, which I think should cover this case.

To make it easier for everyone to compare across languages, it would be good to be consistent.

Reply:
I added an explicit false/0